test/expectedRate.js

```diff
@@ -721,6 +721,27 @@ contract('ExpectedRate', function(accounts) {
 
         let burnerkncRateAfterSet = await tempFeeBurner.kncPerEthRatePrecision();
         assert.equal(burnerkncRateAfterSet.valueOf(), burnerInitialkncRate.valueOf());
+      });
+
+      it("should verify when have no knc arbitrage, set rate success", async function() {
```

Comment:
why is this change in this PR?
I don't follow

Reply:
Was an old issue related to buidler. Updated to use the current expectedRate test file in the `development` branch.